doc/integrator/fulltext_search.rst

```diff
@@ -10,34 +10,33 @@ The *text search* feature requires a dedicated PostgreSQL table. The full-text
 search table should be named ``tsearch`` (for *text search*) and should be in
 the application-specific schema.
 
-You do not need to create the table yourself, as it is created by the
-``create_db`` command line (see the section
-:ref:`integrator_install_application`).
+You do not need to create the table yourself, as it is created by the database creation
```

Comment:
"as it is created by the database creation" -> "as it was already created during application installation"